graybatConfig.cmake

```diff
@@ -44,41 +43,76 @@ set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14")
 ###############################################################################
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${graybat_DIR}/include/graybat/utils/cmake/modules/" )
 
+
 ###############################################################################
 # DEPENDENCIES
 ###############################################################################
 
 ###############################################################################
+# Conan 
+# - Resolves dependencies of ZMQ
+###############################################################################
+if(EXISTS "${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
+  include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
+  conan_basic_setup()
+endif()  
+
+
+###############################################################################
 # METIS LIB
 ###############################################################################
 find_package(METIS MODULE 5.1.0)
-set(graybat_INCLUDE_DIRS ${graybat_INCLUDE_DIRS} ${METIS_INCLUDE_DIRS})
-set(graybat_LIBRARIES ${graybat_LIBRARIES} ${METIS_LIBRARIES})
+if(METIS_FOUND)
+  set(graybat_INCLUDE_DIRS ${graybat_INCLUDE_DIRS} ${METIS_INCLUDE_DIRS})
+  set(graybat_LIBRARIES ${graybat_LIBRARIES} ${METIS_LIBRARIES})
+else()
+  message(ERROR " METIS not found (not necessary).")
+endif()
+
 
 ###############################################################################
 # ZMQ LIB
 ###############################################################################
 find_package(ZMQ MODULE 4.0.0)
-set(graybat_INCLUDE_DIRS ${graybat_INCLUDE_DIRS} ${ZMQ_INCLUDE_DIRS})
-set(graybat_LIBRARIES ${graybat_LIBRARIES} ${ZMQ_LIBRARIES})
+if(ZMQ_FOUND)
+  set(graybat_INCLUDE_DIRS ${graybat_INCLUDE_DIRS} ${ZMQ_INCLUDE_DIRS})
+  set(graybat_LIBRARIES ${graybat_LIBRARIES} ${ZMQ_LIBRARIES})
+else()
+  message(FATAL_ERROR " ZMQ not found.")
+endif()
+  
 
 ###############################################################################
 # Boost LIB
 ###############################################################################
 find_package(Boost 1.56.0 MODULE COMPONENTS mpi serialization REQUIRED)
-set(graybat_INCLUDE_DIRS ${graybat_INCLUDE_DIRS} ${Boost_INCLUDE_DIRS})
-set(graybat_LIBRARIES ${graybat_LIBRARIES} ${Boost_LIBRARIES})
+if(Boost_FOUND)
+  set(graybat_INCLUDE_DIRS ${graybat_INCLUDE_DIRS} ${Boost_INCLUDE_DIRS})
+  set(graybat_LIBRARIES ${graybat_LIBRARIES} ${Boost_LIBRARIES})
+else()
```

Comment:
actually you should never get this branch since `find_package` on `Boost` is already issuing a standardized `FATAL_ERROR` message on `REQUIRED`.